modules/virt-configuring-secondary-dns-server.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-accessing-vm-secondary-network-fqdn.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-secondary-dns-server_{context}"]
+= Configuring DNS server for secondary networks
+
+The Cluster Network Addons Operator (CNAO) deploys the Domain Name Server (DNS) server and monitoring components when you enable the `KubeSecondaryDNS` feature gate in the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You have access to an {product-title} cluster with `cluster-admin` permissions.
+
+.Procedure
+. Create a `LoadBalancer` service to expose the DNS server outside the cluster. The service listens on port 53.
```

Comment:
Sorry, forget it previously. The `target-port` should be 5353. (port 53, target-port 5353)